### DISCUSSION TITLE
DietPi-Software | Tailscale

### DISCUSSION THREAD

**Discussion by cocoflan** [MichaIng/DietPi#5021] [Software Request] [answered]
A secure network that just works

Zero config VPN. Installs on any device in minutes, manages firewall rules for you, and works from anywhere.

**What is Tailscale?**

Tailscale is a VPN service that makes the devices and applications you own accessible anywhere in the world, securely and effortlessly. It enables encrypted point-to-point connections using the open source WireGuard protocol, which means only devices on your private network can communicate with each other.
The Benefits

Building on top of a secure network fabric, Tailscale offers speed, stability, and simplicity over traditional VPNs.

Tailscale is fast and reliable. Unlike traditional VPNs, which tunnel all network traffic through a central gateway server, Tailscale creates a peer-to-peer mesh network (called a tailnet): 

https://github.com/tailscale/tailscale
https://tailscale.com/

**Comment by cocoflan** [comment]
I am testing the free account, up to 20 devices, works on all major platforms also ARM and smartphones.

**Reply by harrydaws** [reply]
Did you manage to get it to work on diet pi?
If so how did you install it (hopefully headless)?

**Reply by Joulinar** [reply]
maybe this link is helpful https://github.com/MichaIng/DietPi/issues/5072#issuecomment-992807749

**Comment by Joulinar** [comment]
Many thanks for your request. I moved it to discussions where we start to collect software requests now. Users can vote here, which is one benefit, since FeatHub is sadly down, respectively unmaintained and cannot be used anymore.

**Comment by MichaIng** [comment]
Simple install via official APT repository, including even a Bookworm suite already: https://pkgs.tailscale.com/stable/#debian-bullseye
```
curl -sSfL 'https://pkgs.tailscale.com/stable/debian/bullseye.noarmor.gpg' -o /etc/apt/trusted.gpg.d/tailscale.gpg
echo 'deb https://pkgs.tailscale.com/stable/debian bullseye main' > /etc/apt/sources.list/tailscale.list
apt update
```

**Comment by Joulinar** [comment, marked as answer]
Will be part of DietPi v8.6

**Reply by Balumis** [reply]
I have already installed, but how can i open it (GUI) ?
On with port will work Tailscale and ZeroTier?

**Reply by Joulinar** [reply]
there is no Gui. Have a look to our online docs https://dietpi.com/docs/software/vpn/#tailscale

As well there is no port that would need to be open.
